frontend/src/app/modules/fields/edit/field-types/multi-select-edit-field.component.ts

```diff
@@ -34,14 +34,15 @@ import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.compon
 import {ValueOption} from "core-app/modules/fields/edit/field-types/select-edit-field.component";
 import {NgSelectComponent} from "@ng-select/ng-select";
 import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
+import {InjectField} from "core-app/helpers/angular/inject-field.decorator";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/0d5d295fec5e57b4542a818c3f681c2be34b856b/frontend/src/app/modules/fields/edit/field-types/multi-select-edit-field.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>